galaxy.yml

```diff
@@ -26,20 +51,30 @@ tags:
   - zos_ims
   - z_and_cloud_modernization_stack_operator
   - zoscb
+
+# Collections that this collection requires to be installed for it to be usable. The key of the dict is the
+# collection label 'namespace.name'. The value is a version range here: https://python-semanticversion.readthedocs.io/en/latest/#requirement-specification. Multiple version
+# range specifiers can be set and are separated by ','
 dependencies:
   ibm.ibm_zos_core: '>=1.3.5'
```

Comment:
Does the core collection version need to updated to match the `requirements.yml`?